release-content/0.14/release-notes/13417_implement_the_full_set_of_sort_methods_on_QueryIter.md

```diff
@@ -1 +1,81 @@
-TODO
+Currently, query iteration order is not guaranteed. If a user wishes to handle their query items in a certain order, sorting is required.
```

Comment:
It would be nice to give an example or two about why in-order iteration is useful.

Reply:
added 2 examples!